docs/tutorials/send_receive_ingress.md

```diff
@@ -43,38 +43,116 @@ Wait for the Operator to start (status: `running`).
 $ kubectl get pod --namespace send-receive-project --watch
 ```
 
-### Deploy the ActiveMQ Artemis Broker
+### Cert manager
+
+#### Install cert manager
+
+```console
+$ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.14.5/cert-manager.yaml
+```
+
+#### Create the root issuer
+
+```console
+$ kubectl apply -f - <<EOF
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: send-receive-root-issuer
+  namespace: send-receive-project
+spec:
+  selfSigned: {}
+EOF
+```
+
+```console
+$ kubectl get issuers  -n send-receive-project -o wide send-receive-root-issuer --watch
+NAME                       READY   STATUS   AGE
+send-receive-root-issuer   True             45s
+```
 
-For this tutorial we need to:
+#### Create the issuer certificate
 
-* have a broker that is able to listen to any network interface. For that we
-  setup an `acceptor` that will be listening on every interfaces on port
-  `62626`.
-* have the ssl protocol configured for the `acceptor`
-* have queues to exchange messages on. These are configured by the broker
-  properties. Two queues are setup, one called `APP.JOBS` that is of type
-  `ANYCAST` and one called `APP.COMMANDS` that is of type `MULTICAST`.
+```console
+$ kubectl apply -f - <<EOF
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: send-receive-issuer-cert
+  namespace: send-receive-project
+spec:
+  isCA: true
+  commonName: ArtemiseCloud send-receive issuer
+  dnsNames:
+    - issuer.artemiscloud.io
+  secretName: send-receive-issuer-cert-secret
+  privateKey:
+    algorithm: ECDSA
+    size: 256
+  issuerRef:
+    name: send-receive-root-issuer
+    kind: Issuer
+EOF
+```
+
+```console
+kubectl get certificates  -n send-receive-project --watch
+NAME                       READY   SECRET                            AGE
+send-receive-issuer-cert   True    send-receive-issuer-cert-secret   24s
+```
 
-#### Create the certs
 
-We'll take some inspiration from the [ssl broker
-setup](https://github.com/artemiscloud/send-receive-project/blob/main/docs/tutorials/ssl_broker_setup.md)
-to configure the certificates.
+#### Create the issuer
 
-> [!NOTE]
-> In this tutorial:
-> * The password used for the certificates is `000000`.
-> * The secret name is `send-receive-sslacceptor-secret` composed from the broker
->   name `send-receive` and the acceptor name `sselacceptor`
+```console
+$ kubectl apply -f - <<EOF
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: send-receive-issuer
+  namespace: send-receive-project
+spec:
+  ca:
+    secretName: send-receive-issuer-cert-secret
+EOF
+```
```

Comment:
Why is the second level issuer required?

Reply:
I mainly followed this unit test to create the tutorial https://github.com/artemiscloud/activemq-artemis-operator/blob/main/controllers/activemqartemis_controller_cert_manager_test.go#L184 It's using the two levels of certificates also.

The documentation of cert-manager also does something similar https://cert-manager.io/docs/configuration/selfsigned/

But I can also try to only get a root issuer if you think this is better.

Reply:
If that works the example would be simpler.

Reply:
Unfortunately, using the root issuer doesn't work and produces this error when the client attempts a connection:

```
11:27:11.396 [pool-2-thread-1] ERROR org.apache.activemq.artemis.core.client - AMQ214016: Failed to create netty connection
io.netty.channel.StacklessClosedChannelException: null
	at io.netty.channel.AbstractChannel$AbstractUnsafe.ensureOpen(ChannelPromise)(Unknown Source) ~[netty-transport-4.1.108.Final.jar:4.1.108.Final]
Connection failed::Failed to create session factory
```